### PR TITLE
silence some warnings

### PR DESCRIPTION
**Pull request by FernetMenta**
see title

### CODE REVIEW

File: xbmc/cores/DllLoader/ldt_keeper.c [hunk 1]
```diff
@@ -263,7 +263,7 @@ ldt_fs_t* Setup_LDT_Keeper(void)
   Setup_FS_Segment();
 
   ldt_fs->prev_struct = malloc(8);
-  *(void**)array.base_addr = ldt_fs->prev_struct;
+  array.base_addr = (uintptr_t)(ldt_fs->prev_struct);
```
Comment (garbear) [inline xbmc/cores/DllLoader/ldt_keeper.c]: static_cast please

Comment (fritsch) [reply xbmc/cores/DllLoader/ldt_keeper.c]: That's C code.

Comment (garbear) [reply xbmc/cores/DllLoader/ldt_keeper.c]: ha should have looked closer. ok from me

**Comment by Rechi** [comment]
wrong revision was build
jenkins build this please

**Comment by Rechi** [comment]
test suite failures are unrelated

**Pull request merged**